reloadRecipe.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+echo "Run this script from the same directory as your recipe metadata.json"
```

Comment:
Hi @LSAllenUcroo! I was using a similar variant of this script yesterday and I ran it from the wrong directory.  There was so much output (as it printed every file in my .git directory) that the above message scrolled off the screen.

What do you think about checking for the metadata.json file in the current directory and, if it's not there, printing this message and exiting?

E.g.

```
if [ ! -r "./metadata.json" ] ; then 
   echo "Run this script from the same directory as your recipe metadata.json"
   exit 1
fi
```

The `-r` means is the file readable, btw.

Reply:
Oh, brilliant suggestion, I'll add that. Thanks @timjstewart1024